### ISSUE TITLE
[Bug] Light Theme Detection Broken

### ISSUE THREAD

**Issue by caoccao** [linebender/druid#2408]
I'm building an application on Windows 10 x86_64 with light theme set. However, the application always shows the dark theme and I failed to find a way of correcting it.

![image](https://github.com/user-attachments/assets/ef596214-dc12-4563-89c1-d28e5a0fac8c)

The registry is as follows.

![image](https://github.com/user-attachments/assets/985b9095-42ec-432c-8ed2-6028a0732a87)

The settings is as follows.

![image](https://github.com/user-attachments/assets/85156e6e-11c8-4c84-a86b-d058617c7328)

**Comment by caoccao** [comment]
**Update**

I pointed the dependency to the latest source code. It's now resolve.

```toml
[dependencies]
# druid = "0.8.3"
druid = { path = "../druid/druid" }
```

![image](https://github.com/user-attachments/assets/0a937d72-c480-4815-b837-fb9597a4ac04)

**Comment by xStrom** [comment]
This was indeed fixed in #2378 and will be part of the next release.
